_schemaV2.graphql

```diff
@@ -2648,6 +2648,9 @@ type Artwork implements Node & Searchable & Sellable {
   isComparableWithAuctionResults: Boolean
   isDisliked: Boolean!
   isDownloadable: Boolean
+
+  # Whether this artwork is eligible for an ecommerce transaction
+  isEcommerce: Boolean
```

Comment:
I think this is a raw field. More like 'galley signals intention' if it wants the work to be `buy now`. The `isAcquireable` is the one that clients use to actually see if `buy now` transaction is possible.